test_xdb/scripts/macrodocs.py

```diff
@@ -59,7 +59,7 @@ def macrodocs(macros_folder, docs_file, header_text, prefix=None):
                     in_return,in_args,in_description,in_support = False,False,False,True
 
                 if in_description:
-                    description += line.replace('{#','').replace('#}','')
+                    description += line.replace('{#','').replace('#}','').replace('/*','')
```

Comment:
I think you also need to remove the closing comment string too `*/`

Reply:
I don't think it's necessary since the rest of the doc comments are parsed and reformatted, but we should strip it just in case (might be necessary for single line descriptions or some silly edge case).